scripts/perf_target_scale.py

```
@@ -0,0 +1,809 @@
+class TargetScaleRunner:
+    """Own one tagged app process and one fixture size at a time."""
+
+    def __init__(self, args: argparse.Namespace):
+        self.args = args
+        self.tag = str(args.tag)
+        self.tag_slug = sanitize_tag(self.tag)
+        self.tag_id = sanitize_bundle(self.tag)
+        self.socket_path = pathlib.Path(f"/tmp/cmux-debug-{self.tag_slug}.sock")
+        self.cmuxd_socket_path = pathlib.Path.home() / "Library" / "Application Support" / "cmux" / f"cmuxd-dev-{self.tag_slug}.sock"
+        self.app_path = pathlib.Path(args.app_path).expanduser() if args.app_path else self.default_app_path()
+        self.binary_path = self.app_path / "Contents" / "MacOS" / f"cmux DEV"
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)